src/router.js

```diff
@@ -1,69 +1,128 @@
-        <Link to="/" className="navbar-brand">
-          Home
-        </Link>
-        <button
-          className="navbar-toggler"
-          type="button"
-          data-toggle="collapse"
-          data-target="#navbarNav"
-          aria-controls="navbarNav"
-          aria-expanded="false"
-          aria-label="Toggle navigation"
-        >
-          <span className="navbar-toggler-icon"></span>
-        </button>
-        <div className="collapse navbar-collapse" id="navbarNav">
-          <ul className="navbar-nav">
-            <li className="nav-item">
-              <Link to="/register" className="nav-link">
-                Register Page
-              </Link>
-            </li>
-            <li className="nav-item">
-              <Link to="/loginPage" className="nav-link">
-                Login Page
-              </Link>
-            </li>
-            <li className="nav-item">
-              <Link to="/adminpage" className="nav-link">
-                Admin Page
-              </Link>
-            </li>
-          </ul>
-        </div>
-      </nav>
-      <br />
-      <br name="very professional line breaks" />
-      {/* A <Switch> looks through its children <Route>s and
+  const logout = async e => {
+    e.preventDefault();
+    sendJson('logout', {
+      currentUser
+    });
+  };
+
+  useEffect(() => {
+    loadJson('checkAuth').then(setCurrentUser);
+  }, []);
+  return (
+    <Router>
+      <div>
+        <nav className="navbar navbar-expand-lg navbar-light bg-light">
+          <Link to="/" className="navbar-brand">
+            Home
+          </Link>
+          <button
+            className="navbar-toggler"
+            type="button"
+            data-toggle="collapse"
+            data-target="#navbarNav"
+            aria-controls="navbarNav"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span className="navbar-toggler-icon"></span>
+          </button>
+          <div className="collapse navbar-collapse" id="navbarNav">
+            <ul className="navbar-nav">
+              <li className="nav-item">
+                <Link to="/register" className="nav-link">
+                  Register Page
+                </Link>
+              </li>
+              <li className="nav-item">
+                <Link to="/loginPage" className="nav-link">
+                  Login Page
+                </Link>
+              </li>
+              <li className="nav-item">
+                <Link to="/adminpage" className="nav-link">
+                  Admin Page
+                </Link>
+              </li>
+              <li className="nav-item">
+                <span className="navbar-text">
+                  {currentUser && `Logged in as ${currentUser.email}`}
+                </span>
+              </li>
+              <li className="nav-item">
+                <form className="form-inline">
+                  <button
```

Comment:
Make sure this button only appears up when someone is logged in.

Reply:
Thats what I'm trying to do right now..